examples/apollo-federation/products/dining.ts

```diff
@@ -0,0 +1,9 @@
+import { Directive, Field, ObjectType } from "../../../src";
+import Product from "./product";
+
+@Directive(`@key(fields: "upc")`)
```

Comment:
Oops. These files shouldn't be here. I'll clean them up.

Looks like I also have some linter errors from the update to clean up as well